file-formats/intm/examples/saplvfkd32izflkp5or2uqi1jlc4.intm.json

```diff
@@ -0,0 +1,35 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Model creation test via ADT",
+    "originalLanguage": "en"
+  },
+  "intelligentModel": {
+    "name": "RM_TEST_ADT_959_MOD",
+    "intelligentScenario": "RM_TEST_ADT_959",
+    "modellingContextId": "0405E348470277F0EBDAEF4E3CEC2C31",
+    "modelType": "SAPGENAI",
+    "adapterId": "SAPGENAI.Default",
+    "nativeModelType": "SAPGENAI.Default",
+    "state": "Published",
```

Comment:
This is not in our current scope. Because we also have state INITIAL when we save the metadata. Also, inactive handling is under discussion and it's for our future scope. So, it's difficult to get rid of state as it's tightly integrated with management Fiori application.